docs/docs/userguide/CONFIGURATION.md

```diff
@@ -50,7 +50,7 @@ The value will be used to form the name of some resources including `vpc` and `K
 environment_name = "<YOUR-ENVIRONMENT-NAME>"
```

Comment:
I know it is against the style we have in the file but what about using lowercase `<your-environment-name>`, it could make it more obvious? Just a thought.